### DISCUSSION TITLE
Who's going to get to 1k first?

### DISCUSSION THREAD

**Discussion by lindsayad** [mfem/mfem#3174] [General]
It's been fun as a lead developer on https://github.com/idaholab/moose watching the growth of our fellow open source finite element library MFEM. Who will get to 1000 stars first? 😉 Seems like you all have the edge!

**Comment by v-dobrev** [comment]
It certainly is a close competition! 😁

**Comment by tzanio** [comment]
Congrats @lindsayad and MOOSE team! 🎉

**Reply by lindsayad** [reply]
Haha thank you, I think a company we're working with all of a sudden helped us out. I think you all have us in the long run 😄 Keep up your great work!

**Comment by lindsayad** [comment]
Now it's :tada: to you all for breaching 1k! And you're back to being ahead :smile: Happy coding!
